app/src/main/res/layout/sheet_text_input_date_picker.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The UI doesn't look the same as in designs
![Screenshot_20210416_075022](https://user-images.githubusercontent.com/6140516/114962531-745e2580-9e88-11eb-98bd-a9abda69e18d.png)

![Screenshot_20210416_071946](https://user-images.githubusercontent.com/6140516/114962529-732cf880-9e88-11eb-82b4-b595ad5156ee.png)